spec/Section 3 -- Type System.md

```diff
@@ -1912,3 +1924,36 @@ type ExampleType {
   oldField: String @deprecated(reason: "Use `newField`.")
 }
 ```
+
+
+### @specifiedBy
+
+```graphql
+directive @specifiedBy(
+  url: String!
+) on SCALAR
+```
+
+The `@specifiedBy` directive is used within the type system definition language
+to provide a URL for specifying the behaviour of custom
+scalar definitions. The URL should point to a human-readable specification of
+the data format, serialization, and coercion rules for the scalar. For example,
+a GraphQL system providing a `UUID` scalar might link to RFC 4122,
+or some document defining a reasonable subset of that RFC. If a specification
+URL is present, systems must conform to the described rules. Built-in scalar
```

Comment:
> If a specification URL is present, systems must conform to the described rules.

I don't think this is possible as "must". If you publish a URL tomorrow with a spec and define a custom scalar with it, there's no way my GraphQL system build today will know how to conform to its rules.

Maybe this can be something like "If a specification URL is present, GraphQL services and client tools should conform to its described rules if they are aware of that URL."